doc/lua_api.md

```diff
@@ -6481,6 +6481,11 @@ Environment access
 * `core.get_node_or_nil(pos)`
     * Same as `get_node` but returns `nil` for unloaded areas.
     * Note that even loaded areas can contain "ignore" nodes.
+* `core.get_node_raw(x, y, z)`
```

Comment:
thought: the current implementation of this allows x, y or z to be nil, which is deprecated.

the newly exposed API function could disallow this from the beginning, at the cost of requiring a separate function.

Reply:
Too much bother if we're about to fully disallow nil in a future release anyway IMO.